# ReadRobotState.srv

# Request

---
# Response
sensor_msgs/JointState joint_state
tf2_msgs/TFMessage transform
